packages/trader/src/AppV2/Components/TradeParameters/PayoutPerPoint/payout-per-point-wheel.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import debounce from 'lodash.debounce';
+import { ActionSheet, Text, WheelPicker } from '@deriv-com/quill-ui';
+import { Skeleton } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import type { TV2ParamsInitialValues } from 'Stores/Modules/Trading/trade-store';
+
+type TPayoutPerPointWheelProps = {
+    barrier?: string | number;
+    current_payout_per_point: string;
+    onPayoutPerPointSelect: (new_value: string | number) => void;
+    payout_per_point_list: {
+        value: string;
+    }[];
+    setV2ParamsInitialValues: ({ value, name }: { value: number | string; name: keyof TV2ParamsInitialValues }) => void;
+};
+
+const onWheelPickerScrollDebounced = debounce(
+    (new_value: string | number, callback: TPayoutPerPointWheelProps['onPayoutPerPointSelect']) => callback(new_value),
+    200
+);
+
+const PayoutPerPointWheel = ({
+    barrier,
+    current_payout_per_point,
+    onPayoutPerPointSelect,
+    setV2ParamsInitialValues,
+    payout_per_point_list,
+}: TPayoutPerPointWheelProps) => {
+    const initial_value_ref = React.useRef<string | number>();
+    const selected_value_ref = React.useRef<string | number>(current_payout_per_point);
+
+    const onSave = () => {
+        initial_value_ref.current = selected_value_ref.current;
+        setV2ParamsInitialValues({ value: selected_value_ref.current, name: 'payout_per_point' });
```

Comment:
On changing value in wheelpicker, it shows in the main page (without saving) . maybe something is to be done for it . `setV2ParamsInitialValues` should do the job but maybe there is something to fix.

Reply:
Agree, will fix, thank you